src/mopi/blocks/augmenters/spelling_autocorrect.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, fast: bool):
         self.fast = fast
 
     def load(self) -> None:
-        self.spell = Speller(fast=self.fast)
+        self.model = Speller(fast=self.fast)
```

Comment:
I've changed the names here so I can consistently delete the loaded stuff from the blocks and it was feasible by naming it consistently``model``. 

Now I'm not actually deleting these models anymore, as we are just exporting the ``unloaded`` pipeline, so if you'd like I can reverse these changes.

Reply:
I think that's fine! it'd be great to enforce always loading the model to `.model`, but not sure if we can do that...